lib/validate_partitions.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+package validate_partitions;
+
+use base "opensusebasetest";
+
+use strict;
+use warnings;
+use scheduler 'get_test_suite_data';
+use testapi;
+
+my $test_data = get_test_suite_data();
+
+# If we want to validate Msdos partition
+sub validate_partition_table {
+    my $root_disk = script_output("mount |grep \" / \"");
+    my $expected_table_type;
+    if (length($test_data->{table_type})) {
+        $expected_table_type = $test_data->{table_type} }
+    else {
+        $expected_table_type = (check_var('BACKEND', 's390x')) ? "dasd" : "gpt";
```

Comment:
why just not set test data in corresponding test suite instead of figure it out the value conditionally?

Reply:
That's what I decidede afterwards, as anyway this would not work if we have something like md0p1 as root partition

Reply:
Arr, finally, I think I will go back to that solution, as: 
on qemu, first disk will be named /dev/vda, on some other backends, /dev/sda, and on zVM /dev/dasda1. So it would force me to create as many yaml files or some if conditions...

Reply:
Now doing both, as for btrfs test uite the disk is not defined, for what I was mentioning in previous comment.